### PR TITLE
Issue #SB-21199 fix: added condition for desktop app

### PR DESCRIPTION
**Pull request by Nik720**

### CODE REVIEW

File: ansible/artifacts/sunbird/login/resources/js/login.js [hunk 1]
```diff
@@ -437,7 +437,7 @@ var handleSsoEvent  = () => {
 		const sessionUrlObj = new URL(sessionUrl);
 		if (redirect_uri) {
 			const redirect_uriLocation = new URL(redirect_uri);
-			if (client_id === 'android') {
+			if (client_id === 'android' ||  client_id === 'desktop') {
```
Comment (SMYALTAMASH) [inline ansible/artifacts/sunbird/login/resources/js/login.js:440]: we don't have any client in keycloak named desktop

Comment (SMYALTAMASH) [reply ansible/artifacts/sunbird/login/resources/js/login.js:440]: @HarishGangula @Nik720 is this new keycloak client reviewed by DC for the authentication and authorization along with valid redirect routes?

Comment (Nik720) [reply ansible/artifacts/sunbird/login/resources/js/login.js:440]: @keshavprasadms had created a new KC client for desktop for staging

**Review by HarishGangula — APPROVED** [review]
These changes are to allow desktop app login and session management

File: ansible/artifacts/sunbird/login/resources/js/login.js (hunk 1 above)
Comment (keshavprasadms) [reply ansible/artifacts/sunbird/login/resources/js/login.js:440]: Thats only for testing your code. The design has to be signed off from DC.

**Review by SMYALTAMASH — APPROVED** [review]

**Pull request merged**